backstop_data/engine_scripts/puppet/search/full-text-result-activity.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Page = require('../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, vp) => {
+  const page = new Page(engine, scenario, vp);
+
+  await engine.type('#text', 'Barry Adams');
+  await engine.select('.crm-form-select', 'Activity');
+  await page.clickAndWaitForNavigation('#_qf_Custom_refresh');
+  await page.fixLayoutForTables();
```

Comment:
`fixLayoutForTables` is already being called in `clickAndWaitForNavigation`, so this would be superfluous

Reply:
Right! Missed that. Let me fix this.